pkgs/by-name/sh/shoko/package.nix

```
@@ -0,0 +1,49 @@
+{
+  buildDotnetModule,
+  fetchFromGitHub,
+  dotnet-sdk_8,
+  dotnet-aspnetcore_8,
+  nixosTests,
+  lib,
+  mediainfo,
+  rhash,
+}:
+buildDotnetModule (finalAttrs: {
+  pname = "Shoko";
+  version = "5.0.0";
+
+  src = fetchFromGitHub {
+    owner = "ShokoAnime";
+    repo = "ShokoServer";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-xGdNkuKhfQZkD9Qn1yrOgUaoQGa9R2lxJ9DfauZ6QpI=";
+    fetchSubmodules = true;
+  };
+
+  dotnet-sdk = dotnet-sdk_8;
+  dotnet-runtime = dotnet-aspnetcore_8;
+
+  nugetDeps = ./deps.nix;
+  projectFile = "Shoko.CLI/Shoko.CLI.csproj";
+  dotnetBuildFlags = "/p:InformationalVersion=\"channel=stable\"";
+
+  executables = [ "Shoko.CLI" ];
+  makeWrapperArgs = [
+    "--prefix"
+    "PATH"
+    ":"
+    "${mediainfo}/bin"
+  ];
+  runtimeDeps = [ rhash ];
+
+  passthru.tests.shoko = nixosTests.shoko;
+
+  meta = {
+    homepage = "https://github.com/ShokoAnime/ShokoServer";
```

You could add a changelog

```suggestion
    homepage = "https://github.com/ShokoAnime/ShokoServer";
    changelog = "https://github.com/ShokoAnime/ShokoServer/releases/tag/v${finalAttrs.version}";
```